packages/shared-metrics/src/util/nativeModuleRetry.js

```diff
@@ -188,8 +187,8 @@ function copyPrecompiled(opts, loaderEmitter, callback) {
           `Copying the precompiled build for ${opts.nativeModuleName} ${label} from ${sourceDir} to ${targetDir}.`
         );
 
-        fse
-          .copy(sourceDir, targetDir)
+        fs.promises
```

Comment:
qs: Where is `fs` required? 🤔

Reply:
fs already required in this file.

Reply:
https://github.com/instana/nodejs/pull/1362/files#diff-9bccaad2e8829a033823a10f239e5912eb26e892a573aea25a7b111837aa6786R8

> const { logger: Logger, uninstrumentedFs: fs } = require('@instana/core')